tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,157 @@
+                    </span>
+                  ) : <></>}
+                  <div>
+                    <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                      {`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                    </span>
+                  </div>
+                  <div>
+                    <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Retracted on ${moment(data.retracted_date, 'YYYY-MM-DD').format('ll')}`}>
+                      {`Retracted on ${moment(data.retracted_date, 'YYYY-MM-DD').format('ll')}`}
+                    </span>
+                  </div>
+
+                </>
+              );
+              default: return (
+                <>
+                  {data.doi ? (
+                    <span className="badge badge-info mr-2">
+                      <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                    </span>
+                  ) : <></>}
+
+                  <span className="text-muted" title={`Scheduled Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                    {`Scheduled Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                  </span>
+                </>
+              );
+            }
+          })()}
+        </div>
+        <div className="card-footer">
+          {(() => {
+            switch (publicationType) {
+              case 'draft': return (
+                <Fragment>
+                  <button type="button" onClick={e => handleResume(e, data.id)} className="btn btn-primary btn-sm mr-2 mb-1">
+                    <i className="fa fa-edit mr-1" />
+                    Resume Draft
+                  </button>
+                  <Button onClick={e => handleDelete(e, data.id)} className="btn btn-danger btn-sm mr-2 mb-1">
+                    <i className="fa fa-trash mr-1" />
+                    Delete Draft
+                  </Button>
+                  <button type="button" className="btn btn-info btn-sm mr-2 mb-1">
+                    Mint Doi
+                  </button>
+                  <button type="button" className="btn btn-success btn-sm mb-1">
+                    <i className="fa fa-share-alt mr-1" />
+                    Share
+                  </button>
+                </Fragment>
+              );
+              default: return (
+                <Fragment>
+                  <button onClick={e => handleRetract(e, data.id)} type="button" className="btn btn-danger btn-sm mr-2 mb-1">
+                    <i className="fa fa-exclamation-triangle mr-1" />
+                    Retract
+                  </button>
+                </Fragment>
+              );
+              case 'scheduled': return <span />;
+              case 'retracted': return <span />;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 32 spaces but found 18.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                                case 'retracted': return <span />;
```